Makefile

```diff
@@ -1,7 +1,15 @@
-obj-m += vwifi.o
+TARGET_MODULE := vwifi
+obj-m := $(TARGET_MODULE).o
+
+ccflags-y := -std=gnu99 -Wno-declaration-after-statement
+KDIR := /lib/modules/$(shell uname -r)/build
```

Comment:
Rewrite as `KDIR ?=` instead of `KDIR :=` since we allow users overriding the variable.